contracts/src/subnetregistry/RegisterSubnetFacet.sol

```diff
@@ -58,7 +58,7 @@ contract RegisterSubnetFacet is ReentrancyGuard {
         });
 
         // slither-disable-next-line reentrancy-benign
-        subnetAddr = address(new SubnetActorDiamond(diamondCut, _params));
+        subnetAddr = address(new SubnetActorDiamond(diamondCut, _params, msg.sender));
```

Comment:
It looks like the registry will set it to the sender, which seems reasonable, however my impression was that the creator should be able to specify a multisig address if they wanted to, so the power over the federated weights aren't in one hand.

Reply:
Maybe registry should also accept it as a parameter?

Reply:
I understand that requires modifying the Rust code as well, so I'll defer to others for what to do here.

Reply:
I added a `Ownable` contract that handles transfer ownership.